packages/adk/lib/util/space.ts

```diff
@@ -0,0 +1,19 @@
+import fs from 'fs';
+export const ACTION_CONFIG_FILE = '.actionconfig';
+
+export function read_space(): string {
+    let result = '';
+    if (fs.existsSync(ACTION_CONFIG_FILE)) {
```

Comment:
Please default it to some meaningful value. Like '<space-name>' instead of being empty.

Reply:
addressed